x-pack/platform/packages/shared/kbn-cloud-security-posture/common/utils/ui_metrics.ts

```diff
@@ -10,8 +10,8 @@ import type { UsageCollectionSetup } from '@kbn/usage-collection-plugin/public';
 
 export const APP_NAME = 'cloud-security';
 
-export const MISCONFIGURATION_INSIGHT = 'misconfiguration-insight' as const;
-export const VULNERABILITIES_INSIGHT = 'vulnerabilities-insight' as const;
+export const MISCONFIGURATION_INSIGHT = 'misconfiguration-insight-v2' as const;
+export const VULNERABILITIES_INSIGHT = 'vulnerabilities-insight-v2' as const;
```

Comment:
As requested by @CohenIdo, added version to namings so it will be easier to track fixed version telemetry reports